lib/utils/request/src/index.js

```diff
@@ -27,6 +27,16 @@ const omit = _.omit;
 const debug = require('abacus-debug')('abacus-request');
 const edebug = require('abacus-debug')('e-abacus-request');
 
+// The default request options
+let defopts = {
```

Comment:
defopts shouldn't be mutable, the behavior of request.defaults is to return a new instance of request loaded with the defaults, as opposed to changing the defaults for other usages of request, which could be very disruptive in an environment with many unrelated users of the request module